ext/php5/dogstatsd_client.c

```diff
@@ -9,38 +9,47 @@
 ZEND_EXTERN_MODULE_GLOBALS(ddtrace);
 
 #define METRICS_CONST_TAGS "lang:php,lang_version:" PHP_VERSION ",tracer_version:" PHP_DDTRACE_VERSION
+#define DEFAULT_UDS_PATH "/var/run/datadog/dsd.socket"
```

Comment:
the usual env var available on the machine (connecting to a random pod) is the following 
```
STATSD_URL=unix:///var/run/datadog-agent/statsd.sock
```

Reply:
I haven't added configurability of the dogstatsd unix socket here. (I.e. we don't support STATSD_URL here at all.)
It just defaults to the unix socket if there's one.

Unlike with the tracer (DD_TRACE_AGENT_URL), I wasn't aware of a "full" url env var for dogstatsd. So `STATSD_URL` is the proper env var for that?

Reply:
I'm honestly confused by the RFC. This was the only statsd socket I was aware of on a POD. I'll ask around and see if I can figure this one out.

Reply:
The internal statsd configuration is different from the one defined in public documentations.

Reply:
It's now configurable via DD_AGENT_HOST, so can also be used on other non-standard paths.